index.js

```diff
@@ -0,0 +1,153 @@
+/* eslint max-classes-per-file: ["error", 3] */
+import { DateTime } from './modules/date.js';
+import Books from './modules/book.js';
+import Store from './modules/storage.js';
+
+class task {
+  static currentBooks() {
+    const books = Store.getBooks();
+    books.forEach((book) => task.addBookList(book));
+  }
+
+  static addBookList(book) {
+    const list = document.querySelector('#list-book');
+    const row = document.createElement('tr');
+    row.innerHTML = `<td>${book.title}</td>
+      <td>${book.author}</td>
+      <td><a href="#" class="btn btn-danger btn-sm delete">REMOVE</a></td>`;
+    list.appendChild(row);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static showAlert(message, className) {
+    const div = document.createElement('div');
+    div.className = `alert alert-${className}`;
+    div.appendChild(document.createTextNode(message));
+    const container = document.querySelector('.firstContainer');
+    // const form = document.querySelector('#book-input');
+    const navigation = document.querySelector('.navigation');
+    container.insertBefore(div, navigation);
+    setTimeout(() => document.querySelector('.alert').remove(), 3000);
+  }
+
+  static clearField() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+}
+
+const listBook = document.getElementById('listBooks');
+const newBook = document.getElementById('newBooks');
+const contact = document.getElementById('contact');
+const navItems = [listBook, newBook, contact];
+const listSection = document.getElementById('list-section');
+const addSection = document.getElementById('add-section');
+const contactSection = document.getElementById('contact-section');
+const sections = [listSection, addSection, contactSection];
+
+const dateTime = document.querySelector('#dateTime');
+
+function saveActiveNavItemLocally(id) {
+  localStorage.setItem('activeNavItem', id);
+}
+
+document.addEventListener('DOMContentLoaded', task.currentBooks);
+document.querySelector('#book-input').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  if (title === '' || author === '') {
+    task.showAlert('fill in all the fields');
+  } else {
+    const book = new Books(title, author);
+    task.addBookList(book);
+    Store.addBook(book);
+    task.showAlert('book added');
+    task.clearField();
+  }
+});
+
+document.querySelector('#list-book').addEventListener('click', (e) => {
+  task.deleteBook(e.target);
+  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+  task.showAlert('Book Removed');
+});
+
+function displaySection(id) {
```

Comment:
- Let's use arraw functions for all your functions as requested in project requirements